Authentication/LogIn/View/LoginController.swift

```diff
@@ -48,10 +55,12 @@ public final class LoginController: UIViewController {
     init(viewModel: LoginViewModelType,
         loginViewFactory: () -> LoginViewType,
         onRegister: (LoginController) -> (),
-        delegate: LoginControllerDelegateType = DefaultLoginControllerDelegate()) {
+        onRecoverPassword: (LoginController) -> (),
```

Comment:
That is not true because somebody needs to provide the window to the bootstrapper so that same object could have access to the window and pass it to the delegate.